### ISSUE TITLE
Gridsome

### ISSUE THREAD

**Issue by dobromir-hristov** [dobromir-hristov/vuecommunity#37]
This page is still debatable, as it needs to expose enough info, to not duplicate over the official Gridsome website.

**Comment by dobromir-hristov** [comment]
Assigned to Tommy from Gridsome Team.

**Comment by gustojs** [comment]
Since Gridsome is a one purpose thing and it's not as crucial to the ecosystem as Nuxt is at the moment, we can keep it only in Static chapter and make a decision to add separate page later.

**Comment by dobromir-hristov** [comment]
Yeah, waiting on a proper description for the Static Site generators page.
